setup.py

```diff
@@ -77,7 +77,6 @@ def run_tests(self):
         'Flask-Limiter',
```

Comment:
If portal is becoming an external package, should we install it by default?

Reply:
I think we should install it by default. I have to create a proper package for PyPI first though..